GU_ASP.NET_MVC_Core/Les7/ReportService.cs

```diff
@@ -0,0 +1,73 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using TemplateEngine.Docx;
+
+
+
+namespace Les7
+{
+    public sealed class ReportService
+    {
+        public void GenerateReport(WorkPlaceTable workPlaceTable, string output = "")
+        {
+            if (workPlaceTable is null)
+            {
+                return;
+            }
+
+            if (string.IsNullOrWhiteSpace(output))
+            {
+                output = Path.Combine(Directory.GetCurrentDirectory(), "WorkPlaceTable.docx");
+            }
```

Comment:
```suggestion
            if (string.IsNullOrWhiteSpace(output))
            {
                output = Path.Combine(Directory.GetCurrentDirectory(), "WorkPlaceTable.docx");
            }
```
лишнее бессмысленное действие